src/main/webapp/manage-availability.js

```diff
@@ -133,43 +43,45 @@ function addTimeSlotHelper(window) {
     });
 }
 
-/** Function to tell the server to delete a time slot for tutor.  */
-function deleteTimeSlot(window, timeslot) {
-    const params = new URLSearchParams();
-    params.append('year', timeslot.date.year);
-    params.append('month', timeslot.date.month);
-    params.append('day', timeslot.date.dayOfMonth);
-    params.append('start', timeslot.start);
-    params.append('end', timeslot.end);
-
-    fetch('/delete-availability', {method: 'POST', body: params}).then((response) => {
-        //if the tutor is not the current user or not signed in
+/** Creates a calendar with the Charts API and renders it on the page  */
+function createCalendar() {
+    fetch('/manage-availability', {method: 'GET'}).then((response) => {
+        //if the student is not the current user or not signed in
         if(response.redirected) {
             window.location.href = response.url;
-            alert("You must be signed in to edit availability.");
+            alert("You must be signed in to manage availability.");
+            return [];
+        }
+        return response.json();
+    }).then((timeslots) => {
```

Comment:
Keep the message that used to be displayed if the user does not have any available timeslots to manage